src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -361,6 +374,13 @@ else if (text.isEmpty()) {
           passthroughClickListener.onClick(view);
         }
       });
+    } else if (messageRecord.getType() == DcMsg.DC_MSG_WEBXDC) {
+      msgActionButton.setVisibility(View.VISIBLE);
+      msgActionButton.setEnabled(true);
+      msgActionButton.setText("Start…");
+      msgActionButton.setOnClickListener(view -> {
+        WebxdcActivity.openWebxdcActivity(getContext(), messageRecord);
```

Comment:
```suggestion
        if (batchSelected.isEmpty()) {
          WebxdcActivity.openWebxdcActivity(getContext(), messageRecord);
        } else {
          passthroughClickListener.onClick(view);
        }
```

Reply:
currently if you click the "start..." button while in selection mode it opens the app